modules/standard/Sort.chpl

```diff
@@ -518,25 +600,24 @@ proc sort(ref Data: [?Dom] ?eltType, comparator:?rec=defaultComparator)
     compilerError("sort() is currently only supported for 1D rectangular arrays");
 }
 
-
 /*
-   Check if array `Data` is in sorted order
+   Check if array `x` is in sorted order
 
-   :arg Data: The array to verify
-   :type Data: [] `eltType`
+   :arg x: The array to verify
+   :type x: `array`
    :arg comparator: :ref:`Comparator <comparators>` record that defines how the
       data is sorted.
    :returns: ``true`` if array is sorted
    :rtype: `bool`
  */
-proc isSorted(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator): bool {
-  chpl_check_comparator(comparator, eltType);
+proc isSorted(x: [], comparator:? = new DefaultComparator()): bool {
```

Comment:
I think this probably should have `const D = x.domain` (or whatever name you prefer) at the top of the function to manually LICM the domain query

Reply:
There's not a loop, but a local seems fine :)